packages/image_picker/image_picker/README.md

```diff
@@ -158,7 +158,7 @@ encourage the community to build packages that implement
 
 Since the macOS implementation uses `file_selector`, you will need to
 add a filesystem access
-[entitlement][https://docs.flutter.dev/platform-integration/macos/building#entitlements-and-the-app-sandbox]:
+[entitlement][3]:
```

Comment:
These kind of link references are hard to maintain correctly in a long document; please use `[entitlement](inline link)`, as with most of the rest of the README.